src/ui/components/landing_page/LoginContainer.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {Box, Divider, Flex, Heading, useBreakpointValue} from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import GoogleLoginButton from './GoogleLoginButton';
+
+export default function LoginContainer() {
+  const minWidth = useBreakpointValue({
+    base: '100%',
+    sm: '400px',
+  });
+
+  return (
+    <Flex direction="column" gap={16} flexGrow="1" minW={minWidth} data-testid="LoginContainer">
+      <Flex direction="column" alignItems="center">
+        <Heading fontSize="xl" textAlign="center">
+          Existing Users
+        </Heading>
+        <Divider />
+        <Box py={4}>
+          <GoogleLoginButton>Sign in with Google</GoogleLoginButton>
+        </Box>
+      </Flex>
+      <Flex direction="column" alignItems="center">
+        <Heading fontSize="xl" textAlign="center">
+          Create an Account
+        </Heading>
+        <Divider />
+        <Box py={4}>
+          <GoogleLoginButton>Sign up with Google</GoogleLoginButton>
+        </Box>
+      </Flex>
+    </Flex>
+  );
+}
```

Comment:
I am not entirely certain we will need a sign up AND sign in button. Might just be a "Proceed with Google" and we can do the OAuth, check if they already exist, and log in if so, otherwise create account.